Assets/BossRoom/Scripts/Game/Server/Action/TossAction.cs

```diff
@@ -0,0 +1,107 @@
+using UnityEngine;
+using BossRoom.Scripts.Shared.Net.NetworkObjectPool;
+using Unity.Netcode;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Action responsible for creating a physics-based thrown object.
+    /// </summary>
+    public class TossAction : Action
```

Comment:
There seems to be some overlap with the LaunchProjectileAction. Could we simplify this by inheriting from it?

Reply:
LaunchProjectileAction was strongly referencing some arrow-specific methods and not all methods were overridable. We certainly could, but it would require modifying LaunchProjectileAction, but with possible upcoming changes to actions, I figured it'd be okay to leave as a standalone action since the actual logic was quite simple.